uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences-my-leave-report.js

```diff
@@ -34,10 +34,10 @@ module.exports = (function () {
       var casper = this.casper;
 
       casper.then(function () {
-        casper.waitForSelector('tr:nth-child(1)  div[uib-dropdown] a:nth-child(1)', function () {
-          casper.click('div:nth-child(2) > div > table > tbody > tr:nth-child(' + (row || 1) + ')  div[uib-dropdown] a:nth-child(1)');
-        });
-      });
+        return casper.waitForSelector('tr:nth-child(1)  div[uib-dropdown] a:nth-child(1)');
+      }).then(function () {
+        casper.click('div:nth-child(2) > div > table > tbody > tr:nth-child(' + (row || 1) + ')  div[uib-dropdown] a:nth-child(1)');
```

Comment:
can the selector be `'div:nth-child(2) table > tbody > tr:nth-child(' + (row || 1) + ')  div[uib-dropdown] a:nth-child(1)'` ? The `div > div > table` selector can be very fragile

Reply:
Done.